.github/workflows/edge-test.yml

```
@@ -80,14 +80,17 @@ jobs:
         run: just prepare-data
 
       - name: 🦀 Check Rust
+        if: false # TODO: re-enable once core2 0.4.0 yank (libflate transitive dep) is resolved
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)